tools/wptrunner/wptrunner/executors/platformaccessibility/executorwindowsaccessibility.py

```diff
@@ -0,0 +1,149 @@
+# mypy: allow-untyped-defs
+
+from ..platformaccessibility.ia2.constants import *  # noqa: F403
+
+import time
+
+import ctypes
+from ctypes import POINTER, byref
+from ctypes.wintypes import BOOL, HWND, LPARAM
+
+import comtypes.client
+from comtypes import IServiceProvider
+
+CHILDID_SELF = 0
+OBJID_CLIENT = -4
+
+user32 = ctypes.windll.user32  # type: ignore
+oleacc = ctypes.oledll.oleacc  # type: ignore
+oleacc_mod = comtypes.client.GetModule("oleacc.dll")
+IAccessible = oleacc_mod.IAccessible  # noqa: N816
+
+# CoCreateInstance of UIA also initializes IA2

```

Comment:
nit: This isn't at all important, but FWIW, I don't think this comment is correct. Importing comtypes initialises COM for this thread. COM is required for both IAccessible and UIA. CoCreateInstance for UIA shouldn't have any impact on IAccessible.

Reply:
Thanks I didn't know! I remove this comment... not sure how I originally came to that conclusion.